### PR TITLE
Semantics to improve readability

### PR DESCRIPTION
**Pull request by josojo**
Following renaming were done:

`sellToken` = `auctionedToken`
`buyToken` = `biddingToken`

and generall the` sellAmount` and `buyAmount` of the auctioneer was better declared by `autioneerBuyAmount`,  `autioneerSellAmount`

Still, I am open to better suggestions and further renamings. SURE!

### CODE REVIEW

File: contracts/EasyAuction.sol [hunk 1]
```diff
@@ -75,12 +75,12 @@ contract EasyAuction is Ownable {
...
-        IERC20 sellToken;
-        IERC20 buyToken;
+        IERC20 auctioningToken;
+        IERC20 biddingToken;
         uint256 auctionEndDate;
         bytes32 initialAuctionOrder;
-        uint256 minimumParticipationBuyAmount;
-        uint256 interimSellAmountSum;
+        uint256 minimumBiddingAmount;
+        uint256 interimSumBiddedAmount;
```
Comment (fedgiac) [inline contracts/EasyAuction.sol]: Unfortunately, the past form of bid is confusingly still bid. (This change applies also to the rest of the file.)

```suggestion
        uint256 interimSumBidAmount;
```

File: contracts/EasyAuction.sol [hunk 2]
```diff
@@ -265,12 +264,12 @@ contract EasyAuction is Ownable {
         (, uint96 buyAmountOfIter, uint96 sellAmountOfIter) =
             iterOrder.decodeOrder();
         require(
-            sumSellAmount.mul(buyAmountOfIter) <
-                sellAmount.mul(sellAmountOfIter),
+            sumBiddedAmount.mul(buyAmountOfIter) <
+                auctioneerSellAmount.mul(sellAmountOfIter),
```
Comment (fedgiac) [inline contracts/EasyAuction.sol]: I think this lines really improves after these changes.

File: src/tasks/initiate_new_auction.ts [hunk 3]
```diff
@@ -8,17 +8,20 @@ import { getEasyAuctionContract } from "./utils";
 const initiateAuction: () => void = () => {
   task("initiateAuction", "Starts a new auction")
```
Comment (fedgiac) [inline src/tasks/initiate_new_auction.ts:9]: Unrelated to this PR, but having a task to start an auction makes a lot of sense!

**Review by fedgiac — APPROVED** [review]
I like the change! IIt's good to separate the "sell/buy token" concept from that of the auction tokens. The new naming works for me, in any case if needed we should be able to change it more easily now.

File: contracts/EasyAuction.sol (hunk 1 above)
Comment (josojo) [reply contracts/EasyAuction.sol]: Oh no... thanks!

**Pull request merged**